test/fixtures/source-map/throw-async.ts

```diff
@@ -0,0 +1,13 @@
+const message = 'Message ' + Math.random();
+
+export async function Throw() {
+  throw new Error(message)
+}
+
+await Throw();
+
+// To recreate:
+//
+// npx tsc --module esnext --target es2017 --outDir test/fixtures/source-map --sourceMap test/fixtures/source-map/throw-async.ts
```

Comment:
If the target is es2018 or higher tsc will not down-level transpile async functions - so that'd how you'd test the thing